.github/workflows/publish_imgs.yml

```diff
@@ -117,9 +115,6 @@ jobs:
               allImgs="$allImgs, $currImgName"
             fi
 
-            nix-shell --run "cd ./imgs/$currImgName && nix-build"
```

Comment:
We dont need to build the images each time, lets leave that up to `publish-imgs` to decide

Reply:
awesome!

Reply:
So my understanding is that the script, `publish-imgs` will take care of that? I like that approach much better. 😃 .

Reply:
Exactly!